.github/workflows/integration_tests.yaml

```diff
@@ -23,7 +23,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-
+      matrix:
+        deps: ["pydantic==1.10.12", "pydantic==2.5.0"]
```

Comment:
1.10.12 is the earliest we tested in our tests before this and 2.5.0 is the first pydantic v2 that we support.